samples/istio/quickstart/B.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  labels:
+    iter8-tools/router: productpage.bookinfo-iter8.svc.cluster.local
+    iter8-tools/role: stable
+  name: bookinfo
+spec:
+  gateways:
+  - mesh
+  - bookinfo-gateway
+  hosts:
+  - productpage
+  - "bookinfo.example.com"
+  http:
```

Comment:
The match section can be simplified based on the actual URLs used. The traffic generation is not manual, but  automated... so we can focus on the URI which is actually used by Fortio (instead of listing all).

Reply:
@sriumcp: Not listing the URLs will break the application, which will show if someone tries to access it from the browser. There might be other consequences to the downstream calls made by productpage.